session/filesync/filesync.go

```diff
@@ -258,11 +259,26 @@ func (sp *fsSyncTarget) DiffCopy(stream FileSend_DiffCopyServer) error {
 	if sp.outdir != "" {
 		return syncTargetDiffCopy(stream, sp.outdir)
 	}
-	if sp.outfile == nil {
+
+	if sp.f == nil {
 		return errors.New("empty outfile and outdir")
```

Comment:
is the error message still correct? (given there's no more `outfile` variable)

Reply:
these vars are private anyway. doesn't change the meaning on the api